examples/step-89/step-89.cc

```diff
@@ -350,6 +350,8 @@ namespace Step89
       Assert(material_id_map.size() > 0,
              ExcMessage("No materials given to MaterialHandler"));
 
+      // PM: no homogeneous path?
```

Comment:
The remote material handler is not used in the homogenous case. The reason is, that we can simply get the materials from the `MaterialHandler`. I will add a comment to the description of the class.